distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/AbstractIntegrationTest.java

```diff
@@ -387,16 +387,18 @@ protected UrlResourceReaderConfigurator getUrlResourceReaderConfigurator() {
     }
 
     protected Option[] configurePaxExam() {
-        return options(logLevel(LogLevelOption.LogLevel.INFO),
-                useOwnExamBundlesStartLevel(100),
+        return options(logLevel(LogLevelOption.LogLevel.INFO), useOwnExamBundlesStartLevel(100),
                 // increase timeout for CI environment
-                systemTimeout(TimeUnit.MINUTES.toMillis(10)),
-                when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-                cleanCaches(true));
+                systemTimeout(TimeUnit.MINUTES.toMillis(10)), when(Boolean.getBoolean(
+                        "keepRuntimeFolder")).useOptions(keepRuntimeFolder()), cleanCaches(true));
     }
 
     protected Option[] configureAdditionalBundles() {
         return options(junitBundles(),
+                wrappedBundle(mavenBundle("ddf.catalog.registry",
```

Comment:
These should not be added as wrapped bundles - this is a flag that you have not created your bundles correctly.